resources/assets/components/Action/index.js

```diff
@@ -86,7 +86,8 @@ class Action extends React.Component {
             </li>
           </ul>
         </div>
-        {new Date(format(parse(campaign.start_date), 'M/d/YY')) > Date.now() ? (
+        {new Date(format(parse(campaign.start_date), 'MM/D/YYYY')) >
```

Comment:
This is parsing a string into a Date, formatting into a string, and then once again into a Date. Instead:

```js
parse(campaign.start_date) > Date.now()
```

Reply:
oh right - updated!